angular.json

```diff
@@ -3,7 +3,7 @@
 	"version": 1,
 	"newProjectRoot": "projects",
 	"projects": {
-		"@deliverysolutions/ngx-intl-tel-input-app": {
+		"ngx-intl-tel-input-app": {
```

Comment:
Why change the project name?

Reply:
It's root level app to test package in projects folder. Without this root level testing app does not start